.gitignore

```diff
@@ -11,3 +11,4 @@ tmp/
 vendor/
 *.log
 .fixtures.yml
+*.zip
```

Comment:
Added because local tests are creating a "rclone.zip" file at the root of the repository. Will be fixed in a later PR